distribution/integ-test-zip/build.gradle

```diff
@@ -35,5 +37,14 @@ publishing {
   }
 }
 
+Task integTestNetty4 = tasks.create("integTestNetty4", RestIntegTestTask) {
```

Comment:
The `numNodes = 2` required a sneaky change to distribution/build.gradle as well.